src/source/Ice/TurnConnection.c

```diff
@@ -57,7 +57,6 @@ STATUS createTurnConnection(PIceServer pTurnServer, TIMER_QUEUE_HANDLE timerQueu
     pTurnConnection->allocationExpirationTime = INVALID_TIMESTAMP_VALUE;
     pTurnConnection->nextAllocationRefreshTime = 0;
     pTurnConnection->currentTimerCallingPeriod = DEFAULT_TURN_TIMER_INTERVAL_BEFORE_READY;
```

Comment:
Don't you need to initialize the entries in the TurnPeer array now? If MEMCALLOC takes care of it, could you please comment it? Though, from what I understand, you might need to set the status of the turn peer right?

Reply:
initially there are zero peers so the value isnt used anyways. Once a peer get added, its status is set properly